sql_generators/active_users_aggregates_v3/templates/view.sql

```diff
@@ -3,7 +3,7 @@ CREATE OR REPLACE VIEW
   `{{ project_id }}.{{ app_name }}.active_users_aggregates`
 AS
 SELECT
-  * EXCEPT (app_version, app_name),
```

Comment:
if these fields are not needed, should they just be dropped from the table itself?

Reply:
Yes and likely will do. The problem is when the queries apply a SUM. We're checking if this data is available somewhere else before removing it from the table.